### ISSUE TITLE
Add CMS

### ISSUE THREAD

**Issue by gmmcal** [gmmcal/gmmcal.com.br#56]
Investigate available CMS or create a custom one to manage all possible parts of data. The requirements are:

* Easy to integrate with Devise.
* Ease to create new models and integrate with UI
* Able to support multiple languages

**Comment by gmmcal** [comment]
After evaluate some CMS available in the market, I've decided to build a custom one for myself. The reasons goes from difficulty to start having something working, integration with current (and future custom frontend) or not supporting latest ruby/rails versions.

**Comment by gmmcal** [comment]
After closing this issue, I discovered [Contentful](https://www.contentful.com, which is a CMS as a service. After checking it, I decided to use as my CMS and not develop one. This way, I wouldn't have to deal with a backend UI development that could be a target for attacks.

They provide a REST API to read information from their service, along with a gem for ruby development and lot's of integration kits for other languages <3.

Still, I've decided to develop my side of code with AR models just like I was going to build before, and create a lib to integrate with their service. If, for some reason, I decide to drop their service and build my own, I only have to remove their lib and build whatever I want.
